src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsTests/CustomNode.tsx

```diff
@@ -0,0 +1,49 @@
+import {Handle, Position} from 'reactflow';
+
+import {ClockCircleOutlined} from '@ant-design/icons';
+import {Tooltip} from 'antd';
+
+import {ExecutorIcon} from '@atoms';
+
+import {Text} from '@custom-antd';
+
+import {LocalStep} from '@models/testSuite';
+
+import {DotsDropdown} from '@molecules';
+
+import {TestNodeContainer} from './SettingsTests.styled';
+
+type CustomNodeProps = {
+  data: LocalStep & {
+    group: number | string;
+    removeNode: (id: string, col: number | string) => void;
+  };
+  id: string;
+};
+
+const CustomNode: React.FC<CustomNodeProps> = props => {
```

Comment:
Shouldn't it be `TestNode`?

Reply:
It is covering delay also, but I agree that this name is bad. Maybe something like `StepNode`?

Reply:
Is it really covering a delay? It's imported later as `TestNode`. Maybe worth to split these two then

Reply:
I already renamed it. Yes it is covering delay, doesn't make sense for me to duplicate this component just to have fine names

Reply:
Not to have a fine name, but to avoid code complexity, and have more clear code. It's minor though and can be fixed later, so resolving.